wouso/interface/cpanel/urls.py

```diff
@@ -75,7 +75,8 @@
     url(r'^races/all/$', 'wouso.interface.cpanel.views.races_groups', name='races_groups'),
     url(r'^races/add/$', 'wouso.interface.cpanel.views.races_add', name='races_add'),
     url(r'^races/groups/add/$', 'wouso.interface.cpanel.views.group_add', name='group_add'),
-
+    url(r'^races/race_can_play_toggle/(?P<id>\d*)/$', 'wouso.interface.cpanel.views.race_toggle_can_play', name='race_toggle_can_play'),
```

Comment:
@catalin-oancea, it would make more sense to for the URL to be 'races/(?P<id>\d*)/can_play_toggle. You are performing the 'can_play' property toggle on a race, not the race id on the race_can_play_toggle.